infra/modules/monitoring/variables.tf

```diff
@@ -0,0 +1,28 @@
+variable "service_name" {
+  type        = string
+  description = "List of services running within ECS cluster"
+}
+
+variable "load_balancer_arn_suffix" {
+  type        = string
+  description = "The ARN suffix for use with CloudWatch Metrics."
+}
+
+variable "high_http_target_5xx_error_count_threshold" {
+  type        = number
+  default     = null
+  description = "High http service 5XX error count threshold"
+}
+
+variable "high_http_elb_5xx_error_count_threshold" {
+  type        = number
+  default     = null
+  description = "High http ELB 5XX error count threshold"
+}
+
+variable "high_target_response_time_threshold" {
+  type        = number
+  default     = null
+  description = "High service latency threshold"
+}
```

Comment:
to keep things simple, could we just have the inputs be service_name and load_balancer_arn_suffix, and remove the rest?

i think we can just set default alerts. and if projects need to customize, they might need to customize more than just the thresholds anyways, like they might need to customize the period, evaluation_periods, datapoints_to_alarm, etc.


so for now let's not give the ability to customize, and we can add customization later if need be.